openassessment/xblock/static/css/openassessment-ltr.css

```diff
@@ -1,3 +1,3 @@
ata:last-child,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata:last-child,.step--student-training .student-training__assessment .question__answers .wrapper--metadata:last-child{margin-right:0}}@media screen and (min-width: 925px) and (max-width: 1299px){.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata,.step--student-training .student-training__assessment .question__answers .wrapper--metadata{float:left;display:block;margin-right:2.35765%;width:65.88078%;margin-bottom:0}.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata:last-child,.step--student-training .student-training__assessment .question__answers .wrapper--metadata:last-child{margin-right:0}}@media screen and (min-width: 1300px){.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata,.step--student-training .student-training__assessment .question__answers .wrapper--metadata{float:left;display:block;margin-right:2.35765%;width:65.88078%;margin-bottom:0}.openassessment .staff-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .wrapper--metadata:last-child,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .wrapper--metadata:last-child,.step--student-training .student-training__assessment .question__answers .wrapper--metadata:last-child{margin-right:0}}.openassessment .staff-assessment__assessment .question__answers .answer__tip,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .answer__tip,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .answer__tip,.step--student-training .student-training__assessment .question__answers .answer__tip,.openassessment .staff-assessment__assessment .question__answers .answer__points,.openassessment .step--peer-assessment .peer-assessment__assessment .question__answers .answer__points,.openassessment .step--self-assessment .self-assessment__assessment .question__answers .answer__points,.step--student-training .student-training__assessment .question__answers .answer__points{display:block}.openassessment .staff-assessment__assessment .question__answers .
```

Comment:
Yeah, it's definitely not ideal - I think it's a case of "it worked this way initially and no one has ever taken the time to fix it".

Ever since I've been working in this repo, I've been treating these giant diffs as "Okay, I see you touched some js and the minified file is different somehow, cool." A while back I added [this check](https://github.com/edx/edx-ora2/blob/master/Makefile#L38) to our travis builds, so you can't pass CI unless the checked-in minified files are correctly updated.